NAMESPACE

```diff
@@ -1,4 +1,25 @@
 # Generated by roxygen2: do not edit by hand
 
 export("%>%")
+importFrom(dplyr,across)
```

Comment:
shouldn't we be also exporting the main package functions here? E.g. `exposure`, `exposure_times`, etc?